[BUG] (enabling CHAMBER_FAN_MODE 0 ends in error)

### Bug Description

CHAMBER_FAN_MODE 0


### Steps to Reproduce

enable CHAMBER_FAN_MODE 0

**Expected behavior:**

compile

**Actual behavior:**

Marlin\src\module\temperature.cpp: In static member function 'static void Temperature::manage_heater()':
Marlin\src\module\temperature.cpp:1219:11: error: expected ';' before 'NOMORE'

#### Additional Information

i think i fixed it with adding ; at the end of line #1211 at temperature.cpp
was: fan_chamber_pwm = CHAMBER_FAN_BASE
edited: fan_chamber_pwm = CHAMBER_FAN_BASE;

[conf.zip](https://github.com/MarlinFirmware/Marlin/files/5758352/conf.zip)

--- a/Marlin/src/module/temperature.cpp
+++ b/Marlin/src/module/temperature.cpp
@@ -1234,7 +1234,7 @@ void Temperature::manage_heater() {
 
         #if ENABLED(CHAMBER_FAN)
           #if CHAMBER_FAN_MODE == 0
-            fan_chamber_pwm = CHAMBER_FAN_BASE
+            fan_chamber_pwm = CHAMBER_FAN_BASE;
           #elif CHAMBER_FAN_MODE == 1
             fan_chamber_pwm = (temp_chamber.celsius > temp_chamber.target) ? (CHAMBER_FAN_BASE) + (CHAMBER_FAN_FACTOR) * (temp_chamber.celsius - temp_chamber.target) : 0;
           #elif CHAMBER_FAN_MODE == 2

